internal/kafka/internal/workers/clusters_mgr.go

```
@@ -505,10 +505,7 @@ func (c *ClusterManager) reconcileDynamicCapacityInfo(cluster api.Cluster) error
 	updatedDynamicCapacityInfo := map[string]api.DynamicCapacityInfo{}
 
 	if c.DataplaneClusterConfig.IsDataPlaneAutoScalingEnabled() {
-		previousDynamicCapacityInfo, err := cluster.RetrieveDynamicCapacityInfo()
-		if err != nil {
-			return err
-		}
+		previousDynamicCapacityInfo := cluster.RetrieveDynamicCapacityInfo()
```

Why do we ignore the error?